apis/status/v1beta1/mutatorpodstatus_types.go

```diff
@@ -0,0 +1,115 @@
+/*
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package v1beta1
+
+import (
+	"strings"
+
+	mtypes "github.com/open-policy-agent/gatekeeper/pkg/mutation/types"
+	"github.com/open-policy-agent/gatekeeper/pkg/operations"
+	"github.com/open-policy-agent/gatekeeper/pkg/util"
+	corev1 "k8s.io/api/core/v1"
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/runtime"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	MutationsGroup = "mutations.gatekeeper.sh"
+)
+
+// MutatorPodStatusStatus defines the observed state of MutatorPodStatus
+type MutatorPodStatusStatus struct {
```

Comment:
MutatorPodStatusStatus sounds a bit weird with the double status. Maybe we could go with MutatorPodState and MutatorPodStateStatus?
(but feel free to ignore this one)

Reply:
It does sound weird, but I think the name was autogenned by Kubebuilder (not that that makes it a good name, but I like to keep things "vanilla" where I can -- not often, sadly -- to make things easier to grok).